packages/rspack-test-tools/tests/configCases/output/library-truthy/index.js

```diff
@@ -0,0 +1,7 @@
+it('should not inject bundlerInfo when library is truthy', () => {
+    const content = require('fs').readFileSync(
+		require('path').resolve(__dirname, "bundle0.js"),
+		"utf-8"
+	);
+    expect(content).not.toContain('__webpack_require__' + '.rv')
```

Comment:
cause the code itself is bundled in test code, so we can't use __webpack_require__.rv directly here

Reply:
Could test it in a hook but current implementation is concise enough (https://github.com/web-infra-dev/rspack/blob/98c0c1357eb36830bf3b0e290fc068899a59a150/packages/rspack-test-tools/tests/configCases/library/modern-module-export-char/rspack.config.js)

Reply:
yes, but it's kind of weird to put test in config hook, because we can't put it('xxx') there, better solution would be we support some placeholder for testing like `__TEST_BUNDLE_CONTENT__` to save fs call here

Reply:
perhaps we can use `test.config.js` to inject some global variables into the runner's global context